docs/README.md

```diff
@@ -76,26 +71,15 @@ you should expect an object that looks like that as the return value of `useProd
 
 ℹ️ To have the full type definition in your development environment, be sure to run `vtex setup` in your project to install all TypeScript types exported by this app.
 
-### `ProductContextProvider`
-
-The `ProductContextProvider` component is capable of providing the `ProductContext` to all of it's children. 
-
-It expects to receive the following props:
-
-```ts
-interface ProductAndQuery {
-  query: Record<string, any>
-  product: MaybeProduct
-}
-```
+### `ProductContext`
```

Comment:
I think we can remove this, what do you think?

Reply:
Agreed, if people start needing it we can make it public.

Reply:
Yep, agreed